ui/tests/acceptance/stepDefinitions/helloContext.js

```diff
@@ -0,0 +1,8 @@
+const {client} = require("nightwatch-api")
+
+const {Given} = require("cucumber")
+
+Given("the user browses to the hello page", function() {
+	return client.page.helloPage().navigateAndWaitTillLoaded();
+})
+
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Too many blank lines at the end of file. Max of 0 allowed.](https://app.codacy.com/gh/owncloud/ocis-hello/pullRequest?prid=5709816)